omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/NameTemplateController2_0Test.java

```diff
@@ -9,64 +9,128 @@
  */
 package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_0;
 
+import java.io.IOException;
 import java.io.InputStream;
 
 import org.apache.commons.io.IOUtils;
-import org.hamcrest.Matchers;
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.webservices.rest.SimpleObject;
-import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.openmrs.module.webservices.rest.web.response.ObjectNotFoundException;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_0.RestConstants2_0;
+import org.openmrs.util.OpenmrsConstants;
 import org.springframework.mock.web.MockHttpServletRequest;
-
-public class NameTemplateController2_0Test extends MainResourceControllerTest {
-
-    @Override
-    public String getURI() {
-        return "nametemplate";
-    }
-
-    @Test
-    public void shouldGetNameTemplate() throws Exception {
-        MockHttpServletRequest req = newGetRequest(getURI());
-        SimpleObject result = deserialize(handle(req));
-
-        String json;
-        try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("nameTemplate.json")) {
-            json = IOUtils.toString(inputStream, "UTF-8");
-        }
-
-        Assert.assertThat(result, Matchers.is(SimpleObject.parseJson(json)));
-    }
-
-    @Override
-    public String getUuid() {
-        return null;
-    }
-
-    @Override
-    public long getAllCount() {
-        return 0;
-    }
-
-    @Override
-    public void shouldGetAll() throws Exception {
-
-    }
-
-    @Override
-    public void shouldGetRefByUuid() throws Exception {
-
-    }
-
-    @Override
-    public void shouldGetDefaultByUuid() throws Exception {
-
-    }
-
-    @Override
-    public void shouldGetFullByUuid() throws Exception {
-
-    }
-
+import org.springframework.mock.web.MockHttpServletResponse;
+
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.iterableWithSize;
+
+public class NameTemplateController2_0Test extends RestControllerTestUtils {
+	
+	private static final String URL_GLOBAL_NAMETEMPLATE = "layout.name.format";
```

Comment:
Why not use this? https://github.com/openmrs/openmrs-core/blob/2.0.0/api/src/main/java/org/openmrs/util/OpenmrsConstants.java#L343